c_src/odbcserver.c

```diff
@@ -1544,50 +1518,39 @@ static void encode_column_dyn(db_column column, int column_nr,
             ei_x_encode_ulong(&dynamic_buffer(state), ts->minute);
             ei_x_encode_ulong(&dynamic_buffer(state), ts->second);
             break;
-	case SQL_C_CHAR:
-//      syslog (LOG_INFO, "strlen_or_indptr=%d colsize=%d len=%d buffer=%p",
-//              column.type.strlen_or_indptr,
-//              column.type.col_size,
-//              column.type.len,
-//              column.buffer);
-        if (column.type.strlen_or_indptr <= column.type.col_size)
-        {
-			if binary_strings(state) {
-				 ei_x_encode_binary(&dynamic_buffer(state), 
-						    column.buffer,column.type.strlen_or_indptr);
-			} else {
-				ei_x_encode_string(&dynamic_buffer(state), column.buffer);
-			}
-        } else {
-            /* Let's assume that result length is unknown.
-               extract part by part.
-               handles https://bugs.erlang.org/browse/ERL-421
-
-               ODBC truncates data for LONGTEXT and BLOB
-
-               column.type.col_size bytes of data are already in column.buffer.
-               */
-            char *bufferptr;
-            /* in bytes */
-            int result_len;
-
-            /* writes info into bufferptr and totallen
-               allocates bufferptr */
-            retrive_long_data(column, column_nr, SQL_C_CHAR, state, &bufferptr, &result_len);
-
-			if binary_strings(state) {
-				 ei_x_encode_binary(&dynamic_buffer(state), bufferptr, result_len);
-			} else {
-				ei_x_encode_string(&dynamic_buffer(state), bufferptr);
-			}
 
+    case SQL_C_CHAR: {
+            char *bufferptr = 0;
```

Comment:
I know it means the same shit, but better follow conventions
```suggestion
            char *bufferptr = NULL;
```